test/addons/make-callback-recurse/test.js

```diff
@@ -66,7 +66,7 @@ assert.throws(function() {
 
   setImmediate(common.mustCall(function() {
     for (var i = 0; i < results_arr.length; i++) {
-      assert.equal(results_arr[i],
+      assert.strictEqual(results_arr[i],
                    i,
                    `verifyExecutionOrder(${arg}) results: ${results_arr}`);
```

Comment:
Hmmm...we have a custom lint rule for that. I know I made it lenient to avoid false positives, but it really should have flagged that...Will have to take a look

Anyway, yes, I'll fix the alignment!